src/session.ts

```diff
@@ -639,6 +640,33 @@ export class SessionManager implements Middleware {
         return resolvedPath;
     }
 
+    private getPowerShellVersionLabel(): string {
+        if (this.powerShellExePath) {
+            var powerShellCommandLine = [
+                this.powerShellExePath,
+                "-NoProfile",
+                "-NonInteractive"
+            ];
+
+            // Only add ExecutionPolicy param on Windows
+            if (utils.isWindowsOS()) {
+                powerShellCommandLine.push("-ExecutionPolicy", "Bypass")
+            }
+
+            powerShellCommandLine.push(
+                "-Command",
+                "'$PSVersionTable | ConvertTo-Json'");
+
+            var powerShellOutput = cp.execSync(powerShellCommandLine.join(' '));
```

Comment:
I confirmed this is safe to do, even without OpenSSL installed

Reply:
I'd prefer not to do this synchronously but it would require a much greater refactor to the calling code to deal with a "thenable".